src/styles/layout/_component-grid.scss

```diff
@@ -56,7 +56,19 @@ $breakpoint-accordion-medium-550: 34rem;
 
     @media (min-width: $breakpoint-scale-up) {
       display: grid;
-      list-style: none;
+      grid-template-rows: repeat(4, 1fr);
```

Comment:
We should reduce the height a little here with `grid-template-rows: repeat(3, minmax(0, 0.8fr));`.

Reply:
Consider it done! https://github.com/sparkbox/accessible-components/pull/137/commits/4297d2dfc667f760eaa5d595a8df506453f3945c